hoomd/md/methods/methods.py

```diff
@@ -30,7 +32,104 @@ def _detach_hook(self):
         self._simulation.state.update_group_dof()
 
 
-class NVT(Method):
+class Thermostat(_HOOMDBaseObject):
+    def __init__(self, kT):
+        param_dict = ParameterDict(kT=Variant)
+        param_dict["kT"] = kT
+        self._param_dict.update(param_dict)
+        self._cpp_obj = None
+
+    @hoomd.logging.log(requires_run=True)
+    def thermostat_energy(self):
+        """Energy the thermostat contributes to the Hamiltonian \
+        :math:`[\\mathrm{energy}]`."""
+        return self._cpp_obj.getThermostatEnergy(self._simulation.timestep)
+
+    def _attach_hook(self):
+        pass
+
+
+class ConstantEnergy(Thermostat):
```

Comment:
I get why but it just seems wrong that this is a thermostat. Could we use `None` instead and use logic in Python to pass a `ConstantEnergy` stat back to C++? The current `_BaseHOOMDObject` has hooks that should make this not terribly messy.